src/prototype.rs

```diff
@@ -1322,11 +1357,24 @@ impl<'a> InlineScanner<'a> {
 
     fn to_node_and_ix(&self) -> (usize, usize) {
```

Comment:
What do these numbers represent?

Reply:
This is a good point, and a strong argument can be made to have at least type aliases. The first is a "node index", which is an offset into the `Vec` underlying the tree. The second is a byte offset in the source text.